tutorpod_autoscaling/templates/pod-autoscaling/k8s/hpa.yml

```diff
@@ -32,3 +32,25 @@ spec:
         averageValue: {{ autoscaling["avg_memory"] }}
   {%- endif %}
 {% endfor %}
+{% set autoscaling_config = dict(iter_autoscaling_config().items(), **POD_AUTOSCALING_EXTRA_SERVICES) %}
+{% for service, autoscaling in autoscaling_config.items() if autoscaling.get("enable_keda") and not autoscaling.get("enable_hpa") %}
```

Comment:
Do you know something about running Keda and HPA at the same time? is it recommended?

Reply:
We can use it but they will compete with each other. I will test how they would both work together.